includes/class-contact.php

```diff
@@ -191,6 +192,12 @@ protected function get_fields( array $instance, array $fields = [], $ordered = t
 				'type'           => 'checkbox',
 				'sortable'       => false,
 				'value'          => 'yes',
+				/**
+				 * Filter Google Map default zoom level of 14 to something else.
+				 *
+				 * @since NEXT
+				 */
+				'zoom'           => apply_filters( 'wpcw_widget_contact_map_zoom', 14, $instance ),
```

Comment:
@fjarrett Actually the only reason I'm not doing it is because I'm using late absolute int type casting when printing the value.